pkg/kubelet/disk_manager.go

```diff
@@ -102,11 +104,14 @@ func (dm *realDiskSpaceManager) isSpaceAvailable(fsType string, threshold int, f
 	if fsInfo.Available < 0 {
 		return true, fmt.Errorf("wrong available space for %q: %+v", fsType, fsInfo)
 	}
-
 	if fsInfo.Available < int64(threshold)*mb {
 		glog.Infof("Running out of space on disk for %q: available %d MB, threshold %d MB", fsType, fsInfo.Available/mb, threshold)
 		return false, nil
 	}
+	if fsInfo.InodesFree == 0 {
```

Comment:
Should we define a threshold for inodes as well that is > 0? By the time kubelet detects that there are inodes, the pods on the node are most likely stuck.